src/DataCollector/PheanstalkDataCollector.php

```diff
@@ -55,23 +55,24 @@ public function collect(Request $request, Response $response, \Exception $except
         $defaultPheanstalk = $this->pheanstalkLocator->getDefaultPheanstalk();
 
         // Collect the information
+        /** @var PheanstalkInterface $pheanstalk */
         foreach ($this->pheanstalkLocator->getPheanstalks() as $name => $pheanstalk) {
             // Get information about this connection
             $this->data['pheanstalks'][$name] = [
                 'name'      => $name,
-                'host'      => $pheanstalk->getConnection()->getHost(),
-                'port'      => $pheanstalk->getConnection()->getPort(),
-                'timeout'   => $pheanstalk->getConnection()->getConnectTimeout(),
+//                'host'      => $pheanstalk->getConnection()->getHost(),
+//                'port'      => $pheanstalk->getConnection()->getPort(),
+//                'timeout'   => $pheanstalk->getConnection()->getConnectTimeout(),
```

Comment:
Even if `$pheanstalk` would have a `getConnection()` method, the `Connection` object doesn't have the required getters. Do you think this information is necessary? If yes, I would propose to add it to the `PheanstalkLocator` in the call of `addPheanstalk()`.

Reply:
I don't think this is a required information; just a useful one.